holmes/core/aux_llms/oci_llms.py

```diff
@@ -0,0 +1,194 @@
+from typing import List, Optional, Dict
+import oci
+import os
+import json
+from oci.generative_ai_inference.models import CohereTool, CohereParameterDefinition, CohereToolCall, CohereSystemMessage, CohereUserMessage, CohereToolResult
+from pydantic import BaseModel
+from openai.types.chat.chat_completion_message_tool_call import ChatCompletionMessageToolCall, Function
+
+class Message(BaseModel):
+    content: str
+    tool_calls: Optional[List[ChatCompletionMessageToolCall]] = None
+
+class Choice(BaseModel):
+    message: Message
+
+class LiteLLMResponse(BaseModel):
+    choices: List[Choice]
+
+class ModelSupportException(Exception):
+    pass
+
+class OCILLM:
```

Comment:
maybe make some sort of interface base class so we have something generic for every time we need to expand litellm?